dracut/99emergency-timeout/timeout.sh

```diff
@@ -42,13 +42,17 @@ _prompt_for_timeout() {
         # Print Ignition logs
         cat <<EOF
 -------------------------------------------------------------------------------
-Ignition has failed. Please ensure your config is valid. Note that only Ignition spec
-v3.0.0+ configs are accepted.
-A CLI validation tool to check this called ignition-validate can be downloaded from GitHub:
+Ignition has failed. Please ensure your config is valid. Note that only
+Ignition spec v3.0.0+ configs are accepted.
+
+A CLI validation tool to check this called ignition-validate can be
+downloaded from GitHub:
     https://github.com/coreos/ignition/releases
+
 Here are the Ignition logs:
 EOF
         journalctl -t ignition --no-pager --no-hostname -o cat
+        echo
     fi
```

Comment:
should we add an else to this if statement and just have it print out failed systemd unit info (assuming there is a failed systemd unit) in the case it's not related to ignition? This wouldn't have to be done in this PR, but is an idea to improve things for the future.